src/dll/Paths.cpp

```diff
@@ -37,6 +37,21 @@ std::filesystem::path Paths::GetPluginsDir() const
     return GetRED4extDir() / L"plugins";
 }
 
+std::filesystem::path Paths::GetRedscriptPathsFile() const
+{
+    return GetRED4extDir() / L"redscript_paths.txt";
```

Comment:
Hmmm, this is coded in redscript right? I was thinking to name it something else if not.

Reply:
It can be anything - the path is passed to scc via the command line.

Reply:
Thinking more about it, I think it is a good enough name 🏃 .